src/Nethermind/Nethermind.Init/Steps/InitializeNetwork.cs

```diff
@@ -544,7 +544,7 @@ private async Task InitPeer()
             _networkConfig,
             _api.LogManager);
 
-        string chainName = ChainId.GetChainName(_api.ChainSpec!.ChainId).ToLowerInvariant();
+        string chainName = NetworkId.GetBlockchainName(_api.ChainSpec!.NetworkId).ToLowerInvariant();
```

Comment:
Do you mean we can move it into `NetworkConfig.DiscoveryDns` and remove the code? It may look better. The only issue I see is that if user will define the config from scratch they will forget to set this option for sure and will get slower startup. It's seems internal'ish, with option to override if user knows what they do.